agent/util-scripts/pbench-register-tool

```diff
@@ -116,9 +116,8 @@ function usage() {
 	printf -- "\tdenoted by a leading hash, or pound (\"#\"), character.\n"
 	printf -- "\nAvailable tools:\n"
 	local tool=""
-	for tool in $(find ${pbench_bin}/tool-scripts -maxdepth 1 ! -type d ! -name '*README*' ! -name base-tool ! -name unittests -printf "%P\n" 2> /dev/null | sort); do
-		printf -- "\t${tool}\n"
-	done
+        awk '/null/ {print $1}' ${pbench_bin}/tool-scripts/meta.json | sed 's/"/\t/' | sed 's/"://'
+        awk '/collector/ {print $1}' ${pbench_bin}/tool-scripts/meta.json | sed 's/"/\t/' | sed 's/":/ (persistent)/'
```

Comment:
This is awfully dependent on details, like the fact that we don't have any persistent tool attributes.

This makes me really miss the neat little `jq` utility... although it'd be easy enough if `bench-register-tool` was converted to Python.

```
jq 'keys[] as $k|.[$k]|keys[]' meta.json
```

Reply:
I was originally going to use jq, but it would require additional installation and I wasn't sure if we wanted that. This solution is essentially a placeholder until Chuck's work puts the script into python, in which case I can simply use the ToolMetadata class to retrieve and print tool names. Until then, it should hold true that these properties will be stable for transient/persistent tools, but also open to alternate approaches (I'm not the most well-versed in bash)

Reply:
> I was originally going to use jq, 

Yeah, it was more a side comment than a criticism; converting to Python is a better solution than adding an extra package dependency. I just couldn't resist!